install.sh

```diff
@@ -1317,14 +1316,14 @@ setup_shared_hook_repositories() {
         echo "Shared hook repositories are now unset. If you want to set them up again in the future, run this script again, or change the 'githooks.shared' Git config variable manually."
         echo "Note: shared hook repos listed in the .githooks/.shared file will still be executed"
     elif git config --global githooks.shared "$SHARED_REPOS_LIST"; then
-        # Trigger the shared hook repository checkout manually
-        cp "$GITHOOKS_CLONE_DIR/base-template-wrapper.sh" ".githooks.shared.trigger" &&
-            chmod +x ".githooks.shared.trigger" &&
-            ./.githooks.shared.trigger
-        rm -f .githooks.shared.trigger
-
-        echo "Shared hook repositories have been set up. You can change them any time by running this script again, or manually by changing the 'githooks.shared' Git config variable."
-        echo "Note: you can also list the shared hook repos per project within the .githooks/.shared file"
+
+        "$GITHOOKS_CLONE_DIR/cli.sh" shared update
+
+        echo "Shared hook repositories have been set up. You can change them any time"
```

Comment:
/nit How did you choose the line length here?
Some of them look a bit funny. Maybe the `You can change them any time` for example could start on a new line?